app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -284,6 +322,19 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
 
     @Override
+    protected void onActivityResult(int requestCode, int responseCode, Intent intent) {
+        if (PrefUtils.isSignedIn(this)) {
+            switch (requestCode) {
+                case GdgActivity.RC_SIGN_IN:
```

Comment:
I guess eventually we will need this but I really don't like making `RC_SIGN_IN` public. 
Right now, as an alternative, you can override `onConnected` and do this update there.